### DISCUSSION TITLE
Best scanner to use with paperless?

### DISCUSSION THREAD

**Discussion by Anaxagoras-bc** [paperless-ngx/paperless-ngx#5042] [General]
What scanner do you think is the best to use with paperless?  I want something with a touch screen that i can use without having my pc on, and 1 touch scan to various destinations.  A shared network folder, Cloud drive, ftp/sftp/ssh, etc. 

I've been looking at the usual suspects Fuji/Ricoh scansnap ix1600, brother ADS-4900w or ADS-4700w, Epson WorkForce ES-580W, Cannon, Xerox.

I have an older scansnap, but i dislike their software lock-in and lack of twain drivers.  It's been a great scanner, though, fast, almost never jams, etc.  I'm tempted to give a newer model a try, but i'd also like to make sure that there's not a better option out there.

**Comment by Anaxagoras-bc** [comment]
so i do have an older scansnap as i mentioned.  I decided to try a scanner off of amazon and ordered a brother ads-4900w.

To scan to most cloud drives it looks like you need to have you PC on with the ads-4900w, but i think the Ricoh ScanSnap ix1600 doesn't?  Playing around in the settings i found that it supports webdav, after some trial and error it can scan directly to nextcloud, which is a feature i do have some use for.

Also given it supports scanning to ssh/sftp, or samba/cifs on it's own, scanning directly to a consume folder should also be trivial.

**Reply by tfvlrue** [reply]
Wanted to chime in because I just bought a ADS-4700W to use to upload scanned documents directly to Nextcloud based on the suggestion from this post that it would work. I was a little hesitant, because the scanner's manual doesn't refer to WebDAV (the protocol used to upload to Nextcloud) anywhere, but it does in fact support it!

You have to configure a "SharePoint" profile like so:
- For SharePoint Site Address, use your Nextcloud's WebDAV address (e.g. `https://example.com/remote.php/dav/files/<Username>/<FolderName>`)
- For Username, use your Nextcloud username
- For Password, generate an app password from the Settings > Security section of Nextcloud and use that
- For Auth Method, use Basic or just leave it set to Auto

Hope this helps someone else in the future :)

**Reply by PhilLab** [reply]
Thank you @tfvlrue for this remark, it was exactly what I needed ❤️ . One additional remark for others who might read this: It did not work with the firmware version 2.17. But once I updated to 2.22 using the [firmware update tool](https://support.brother.com/g/b/downloadlist.aspx?&c=de&lang=de&prod=ads4700w_us_eu_as&os=10068&type3=10365&dlid=dlf105550_000), it worked.

**Reply by delprimero** [reply]
For anyone else trying this, I got lost when trying to find the app passwords section in Nextcloud and was banging my head against a virtual wall for an hour or two because I had created an OAuth 2.0 clients password in `settings/admin/security` rather than an app password found under `/settings/user/security#generate-app-token-section`.

**Reply by spulg** [reply]
For me it only worked using `https://example.com/remote.php/dav/files/<Username>/<FolderName>`. I am using the brother ADS-1800W.

**Comment by BlankSlate56** [comment]
I have a Brother 3710.  Its a fantastic unit, can scan direct to network or email via 1 touch on the screen.  Brother support is top notch.  I do wish that it supported duplex scanning to scan 2-sided docs though.

**Reply by noseshimself** [reply]
Especially Brother and their customer orientation is annoying me like no other.

Double-sided scan via pre-set profile? Only if you scan to SMB. This bug is in their software for a decade and although once a month someone is opening a ticket for that (and a few more UI inconsistencies) it was never fixed. And as most printed paper is printed on both sides these days. It means another three (slow!) touch panel presses for every single scan.

No more B(r)other for me.

**Reply by binderth** [reply]
I've got a Brother ADS-1700W, which does double-sided scanning via Email/SMB. I didn't check the USB, because, that's not my usecase.

What's annoying is the "are you sure to send"-Button after scanning the document with a shortcut-button, that I wish could be configured to just send. I'm using shortcuts to reduce clicks... ;)

**Reply by noseshimself** [reply]
>  which does double-sided scanning via Email/SMB.

All double-sided Brother scanners are doing this since the beginning of time. They just forgot to add this to FTP/SSH shortcuts and refuse to take the issue seriously. This is the dictionary definition of idiocy.

**Reply by marcelbrueckner** [reply]
I've got a Brother ADS-1700W, too and it does double-sided scanning to each target, no matter if USB, SMB or FTP. I've defined all settings in the printer's Web UI and scan via shortcut button on the scanner.

**Reply by bshep** [reply]
There is a setting to send without confirmation. Check the settings menus ( if you need it i can reply back where it is when i get home)

**Reply by binderth** [reply]
hey @bshep! 
That would be nice, I did not find the setting... 🙈

**Reply by bshep** [reply]
@binderth Its actually not that obvious: Settings -> General -> Display Scan Results -> Set to 'Off'

Edit: 

Also make sure you turn off the 'continous scan' option in the shortcut itself.

Oh and you can make the 'Home' Screen be the shortcuts screen so you can trigger a scan with just one touch.

**Reply by binderth** [reply]
thanks @bshep works now!

**Reply by mystereit** [reply]
> @binderth Its actually not that obvious: Settings -> General -> Display Scan Results -> Set to 'Off'
> 
> Edit:
> 
> Also make sure you turn off the 'continous scan' option in the shortcut itself.
> 
> Oh and you can make the 'Home' Screen be the shortcuts screen so you can trigger a scan with just one touch.

what exactly does "continous scan" mean/do? couldnt find an explanation anywhere

thank you

**Comment by Joerg-L** [comment]
Epson ES-C380W is doing a good job. No computer required for scanning, double-side scan, option to auto remove empty pages and a touch screen which makes the usage quite easy

**Reply by McGeifer** [reply]
Same here, really good option.

**Comment by Se7enair** [comment]
I use this one https://www.canon.de/business/products/office-printers/multifunction/colour/i-sensys-mf750-series/specifications/i-sensys-mf754cdw.html
Canon i-Sensys MF754CDw

I know it is a combination and not only a single scanner. But for me its the best.
I scans extremely fast, double side scanning in one step, automatic recognition if double side is needed. Can create buttons on the touchscreen to directly scan to the consume folder. And also create different buttons for different setups, e.g. on button to scan a few single sided documents in one run. Or scan a single multi-site document.

And if you take a look at how much very good document scanners cost....

And I love the speed of this thing :sunglasses:

**Reply by mouschd91** [reply]
I have the same printer on my watchlist, the toner is expensive but I hardly print anymore so the starter would be enough for the next 5-8 years. But I hardly find any review of the printer, only heard its fan is pretty loud even in standby, can you confirm this? Noticed any other problems so far?

**Reply by Se7enair** [reply]
> I have the same printer on my watchlist, the toner is expensive but I hardly print anymore so the starter would be enough for the next 5-8 years. But I hardly find any review of the printer, only heard its fan is pretty loud even in standby, can you confirm this? Noticed any other problems so far?

My hope is that the toner will get some third party replacement in the next months/years. It uses a quite new format. 

The printer is Not the quietest. But also not louder than I expect a printer. 
At the moment I do not remember  noises in standby. The printer is placed directly next to my desk. But I will listen more closely as soon as I get home. 

And no, no other problems. Only the settings are a bit complicated. But I am used to epson and hp printers.

**Reply by Se7enair** [reply]
> > I have the same printer on my watchlist, the toner is expensive but I hardly print anymore so the starter would be enough for the next 5-8 years. But I hardly find any review of the printer, only heard its fan is pretty loud even in standby, can you confirm this? Noticed any other problems so far?
> 
> My hope is that the toner will get some third party replacement in the next months/years. It uses a quite new format. 
> 
> The printer is Not the quietest. But also not louder than I expect a printer. 
> At the moment I do not remember  noises in standby. The printer is placed directly next to my desk. But I will listen more closely as soon as I get home. 
> 
> And no, no other problems. Only the settings are a bit complicated. But I am used to epson and hp printers. 

There is a noise when the Printer is in standby. But it has a sleep/deep standby in which it is completely silent.

**Comment by NetRat88** [comment]
Using the Epson ES-580W and really love the speed. Everything you want is included. Touch, scan without PC, WebDAV/SMB, Email etc.

**Reply by tfvlrue** [reply]
Agreed! The paper handling is _much_ better than the Brother ADS-4700W, and it can do some post-processing (like skew correction) when scanning without a PC. The Brother often fed receipts crooked, or not at all, or crumpled them. I'm much happier with the Epson.

One annoying thing I've found with the Epson is that the WebDAV/FTP/SMB passwords are restricted to only 20 characters, which isn't even long enough to hold a Nextcloud app password. I wound up making a WebDAV relay using rclone to work around this limitation.

**Reply by rajil** [reply]
- Does epson 580w have native linux support using SANE driver?
- Can it detect when multiple sheets are fed incorrectly?

**Reply by stewmehr** [reply]
@rajil 
> - Does epson 580w have native linux support using SANE driver?

Based on the [list of supported devices](https://www.sane-project.org/sane-mfgs.html#Z-EPSON) the 580W appears to be fully supported, yes.

**Comment by mquade** [comment]
I'm using an Epson DS-730N. For me LAN-Connectivity was essential. You don't need to install any software or drivers to a computer to set up the scanner. I first tried a Kodak Alaris S2060w. But this was an absolut PITA to set up. I didn't find a way to use it without a dedicated PC. After reading the specs, there must be a way. But after 4-6h of trying I was very frustrated, did put in on ebay and went with the Epson. It is a bit slower but works great. Scanning directly to the import folder works great. Resolution, speed, handling of different paper qualities... all what you expect from a semiprofessional scanner. We imported ~10.000 documents in two weeks. 

I opted for the 730 because of the "speed dail buttons". No touchscreen interaction needed. 
On thing to mention: The dispay is very small and not a touch screen. You need to use the navigation keys. In 2024 this feels very outdated. But to be honest: The users only need the thee speed dials. And configuration is completely done via web frontend.

**Comment by AlexandreBonneau** [comment]
The Brother ADS-2700w works great on Linux, although it tends to forget the date from time to time, resetting it to 2018, which is annoying.

**Comment by akulbe** [comment]
> What scanner do you think is the best to use with paperless? I want something with a touch screen that i can use without having my pc on, and 1 touch scan to various destinations. A shared network folder, Cloud drive, ftp/sftp/ssh, etc.
> 
> I've been looking at the usual suspects Fuji/Ricoh scansnap ix1600, brother ADS-4900w or ADS-4700w, Epson WorkForce ES-580W, Cannon, Xerox.
> 
> I have an older scansnap, but i dislike their software lock-in and lack of twain drivers. It's been a great scanner, though, fast, almost never jams, etc. I'm tempted to give a newer model a try, but i'd also like to make sure that there's not a better option out there.

Did you end up going with the ScanSnap? I just got an ix1600, and I'm wondering if I can use it with paperless-ngx. Trying to see if there are other folks who have had success with this particular device.

**Reply by Anaxagoras-bc** [reply]
I wound up going with the ads-4900.  I had an older scansnap and while it was a good device for the way i wanted to use it, my PC had to be on.  The ADS-4900 while a little slow can work fully intendent of a computer.  One of my wants was to be able to upload a document directly to nextcloud, this can be done through the sharepoint interface.  It can also scan direct to ftp/sftp, and network drives.

**Reply by kdoren** [reply]
I'm very happy with the ScanSnap ix-1600.  I scan (without PC) to a paperless-ngx "consume" SMB share.  Great quality double sided scans.  Excellent paper handling.  It does require a PC to set up though.

I use multiple profiles depending on document type.  Single vs double sided, rotated or not.  On rare occasions I use the ScanSnap home software to delete and rotate individual pages before saving to paperless-ngx.

**Reply by yeraziael** [reply]
Got a Brother ADS-1800W, price was the reason, form factor and design another. After 10 days I finally got to set it up and wasted 3 full days on not getting it to do it's flippin' job and scan to an scp target. Not only did I have to install Samba (no windows devices in my home so that was the only reason to have a sambaserver running) I also had to reduce the security features to standards of the late 90's (no support for RSA with more than 2048bit, wtf?)
Firmwareupdate requires a Windows-PC and a dedicated updating software, did I mention that there's no Windows PC in my home?
Should've done my freaking homework before ordering, like I would do for any customer, but no: if it's for myself it's 'YOLO' - and there goes my weekend.
To be fair, direct Scan to the app worked like a charm.
Sent it back just in time to get a full refund.

Ordered the Epson ES-580W (after doing the homework( RTFM before buying) no nasty surprises this time. Still need the sambaserver, FTP refuses and I ran out of patience, had the samba running anyway. So I stopped fiddling and everything works. My daughter now has the instruction when post arrives to get the scanner (from the office), go to a place where she can put it out of reach for her little brothers and scan the letters (of course I taught her to only operate the scanner after doing the necessary preparation of removing staples, straightening the paper and doing a little scanning dance around the booting device while reciting lines from the Linux kernel source code)

The whole process now reduces to pressing the button for network scan, pressing the button for the predefined scantarget (the consume folder), putting the paper in the right way, pressing scan and then putting the paper on the "scanned" heap.

I am very happy with the Epson es 580w and it's definitely worth the price since the quality of the scans is more than enough for paperless to do it's magic, I can predefine pretty much everything I need and one day the big pile of old papers is gone and then a more expensive scanner won't be justifiable anymore.

Just do yourself 2 favours before you pick your scanner, 
1. have a clear idea of what you need, what you want and what your budget is. 
2. DYOR (do your own research, mainly through RTFM on your preferred manufacturers website) and use your checklist from 1. to make your pick

If something that's vital to you is not mentioned in the official manuals, chances are you need a dirty hack to get there. And then maybe a different printer is a better idea than having to setup an IoT-Scanner on SANE, run by a pi zero w that you fit into the housing after opening it (hopefully without breaking anything) because you wanted a portable scanner that can be stowed away, so that thing cannot have dangling parts that are sensitive AND exposed... but I am trailing off.

Disappointed by brother due to firmware problems (btw. After allowing the old protocols Server side I till didn't get it to work in a reproducible manner, meaning the scanner starts and nondeterministically decides whether it connects only to WiFi, WiFi direct through the scan app or not at all and waits to be restarted and when you get connection it still rolls the dice again to see if any of the email contacts, ftp connections or WebDAV are gonna work today. I mean it's cute but unfortunately useless. Cute doesn't get the job done.

ES 580W is a bit chunky but brings all the features I need and connects to the infrastructure I have. (Am still gonna work out the whole sftp thing but we're in our home network and external connections come only  through VPN, while 19 out of 20 it's me who's connecting, the remaining 1 out of 20 it's me using another family members device. So I guess that simple password protection might just be good enough in this case.)

**Comment by niels-przybilla** [comment]
I am using the Brother ADS2800W. good scanner and scans directly via SFTP into my consume folder.
As far as I read from manuals, the ix1600 cannot scan directly without a computer to SFTP.

I also tested the ix1500 and the scan quality and versatility for scanning smaller documents and empty page detection is in my opinion, better than from the brother.

**Reply by R3dst4r** [reply]
I want to scan to consume with a Brother ADS, too.
How did you set up SFTP? I know about changing the profile to SFTP but authentication fails. Connection information are correct. Using SSH from another machine works fine and uploading into the folder is functional when using another SFTP-client. The scanner is behind a NAT router.

Do you use password or key? I‘m struggling with key-based auth. I checked the manual but wasn‘t able to get it working.

Can you share a generic version of your SFTP-config?

**Reply by wduda** [reply]
Do you have an SFTP server running or only SSH? These are not the same, SFTP is built on top of SSH but you having SSH enabled does not automatically enable SFTP.

**Reply by R3dst4r** [reply]
I'm using https://github.com/atmoz/sftp and FileZilla

**Reply by R3dst4r** [reply]
> Do you have an SFTP server running or only SSH? These are not the same, SFTP is built on top of SSH but you having SSH enabled does not automatically enable SFTP.

I forgot to mention, that I solved my issue. The trick was to create a client key and import the server‘s public key. Add the client‘s public key to the sftp server as a whitelisted client. Using atmoz-sftp, it also supports multiple scanners. The only thing that annoys me a bit is that the initial setup requires some manual tasks (e.g. generating a fresh server key, adding the scanner‘s client key). 

Besides these minor itches, the entire brother ADS series works like a charme! Love these devices!

**Reply by ikluhsman** [reply]
I did this too, same scanner ADS 2800W. I setup an SFTP container in the same docker compose stack as PaperlessNgx and used the same IP as my paperless web server to host the SFTP on 2222. Once that was setup, I created 9 or so SFTP profiles that I could create shortcuts to in the menu on the scanner, and each one refers to a different document type in PaperlessNgx. When a document is consumed, it knows from the filename weather it was business or personal, tags it as such, and it knows from the path name what Document Type was scanned, and it adds that on consume. For the path trick to work you'd need to set [PAPERLESS_CONSUMER_RECURSIVE=true](https://docs.paperless-ngx.com/configuration/#PAPERLESS_CONSUMER_RECURSIVE) and use wildcards to fish out the path you want in your workflow consume rule i.e. ```*/invoices/*```

Then when the document is added (OCR) it tags the rest of the info, correspondent, and sets a sane title for the document.

That's my system and it works very nicely.

**Comment by tmaier** [comment]
I recently purchased both the ScanSnap ix1600 and the Brother ADS-1800w for comparison.

## My Feature Wish List

- Must operate without a PC (supported by both)
- Must support macOS (supported by both)
- Must support scanning to network drives (as fallback): I prefer SFTP over SMB and FTP (supported by both)
- Must handle long receipt scanning
- Must include duplex scanning (supported by both)
- (Optional) Detect when multiple sheets are fed at once
- (Optional) Built-in OCR support
- (Optional) USB-C charging capability
- (Optional) Linux compatibility, as I own a Linux-based laptop
- (Optional) Support scanning to Google Drive Team Drives, as this allows scanner use in multiple locations (homes, offices). I have a Google Drive Team Drive that syncs to my paperless-ngx consume folder on my Synology NAS (broadly supported by both)

## Brother ADS-1800w

- The design and form factor are appealing
- Browser-based configuration is available, though firmware updates require a Windows or Mac PC
- USB-C connectivity is a significant advantage, allowing me to store the scanner on a shelf and connect it to any USB-C cable when needed
- Network drive scanning setup was successful with FTP. SMB likely works as well, but I didn't test it due to its slow speed. My attempt to implement SFTP failed as the ADS-1800w consistently showed connectivity issues, unlike my other devices
- Cloud scanning is supported but seems poorly implemented. The process requires navigating to a dedicated online services section, enduring multiple waiting periods, accepting internet usage terms, and then selecting the cloud service. While it supports multiple Google Drive accounts, it lacks Google Drive Team Drives support
- Long receipt scanning was unsuccessful; I couldn't determine how to enable this feature
- Does not detect when multiple sheets are fed at once
- Offers basic Linux support through some driver packages

## ScanSnap ix1600

- Heavier than the ADS-1800w and lacks USB-C charging
- Requires Windows or Mac PC for initial setup but not for firmware updates
- Excellent cloud scanning capabilities, including Google Drive Team Drives support. Having previously configured Google Drive to Synology NAS synchronization, file transfer to the destination folder is seamless. All workflows can be customized, and the cloud scanning feature includes advanced functions like OCR
- Network drive support is limited to SMB according to documentation (see <https://www.pfu.ricoh.com/imaging/downloads/manual/ss_webhelp/en/help/webhelp/topic/ope_basic_networkfolder.html>)
- Detects when multiple sheets are fed at once
- No official Linux support; Linux compatibility remains untested

While I continue to evaluate scan quality, I'm currently leaning toward the ix1600.

**Reply by reaperharvest** [reply]
@tmaier Can you clarify what you mean by "Does not detect when multiple sheets are fed at once"? The brother seems to have a 20 sheet automatic feeder, so I'm not sure what is missing? Was interested in the ads-1800w before your comment gave me pause.

**Reply by indigoblue79** [reply]
What "Does not detect when multiple sheets are fed at once" means is that if you set 20 sheets in the feeder and fewer than 20 scan, the scanner has the capability to notify you

**Reply by indigoblue79** [reply]
To clarify, I should say ""Does not detect when multiple sheets are fed at once" means "Does not detect fewer than 20 are scanned" in my example.

**Reply by tmaier** [reply]
@reaperharvest what I meant:
Sometimes the ADR will take not one, but multiple sheets into the scanner. This means we would skip a sheet. So instead of lets say 10 sheets scanned, you would just have 9.

**Reply by reaperharvest** [reply]
Got it, thanks for the response!

**Reply by richie80** [reply]
What is your experience until now? Still fan of the ricoh? 
You turn OCR of I presume

**Reply by tmaier** [reply]
Hi @richie80, I am still very happy with ScanSnap ix1600. I just used it a few minutes ago. I have it folded away in a shelf. I take it out whenever I want to digitize my inbox. It connects quickly to the WIFI, scans to the LAN folder for Paperless. The only thing, which I am not fond of is the baked in OCR capabilities. I think they are not that good.

**Reply by binderth** [reply]
AFAIR the OCR recognition within a PDF isn't used by paperless, but its own OCR (via OCRmyPDF atm)?
so it wouldn't matter, if the OCR quality of the scanner is bad...

**Reply by rajil** [reply]
Does ix1600 work with linux using SANE?

**Reply by binderth** [reply]
@rajil should work: http://www.sane-project.org/lists/sane-mfgs-cvs.html

(one reason I chose the ADS-1800w as it can scan directly to a SMB/CIFS folder, which I configured as hotfolder for paperless)

**Reply by rajil** [reply]
@binderth, Thanks. I only have linux systems so that is a major consideration.

Seems Wifi is not support on ix1600 per http://www.sane-project.org/lists/sane-mfgs-cvs.html#Z-RICOH
This is a big deal.

I wonder if there is any scanner which supports,
- Multifeed Detection
- Wifi
- Linux

**Reply by binderth** [reply]
@rajil What's your use cases? 
You won't need an OS in between, if you only just scan paper to paperless...

That's what I do: the ADS-1800w in my case scans pages and sends them directly to a CIFS/SMB share. That share is configured as a hot folder in paperless, which consumes the pdfs (you can also have shortcuts for multiple users, so we use it as a family to scan all our documents - and they get sorted into the correct permissions for each user.

No need for a (Linux) OS on between (only as host for my docker setup for paperless-ngx...)

**Reply by kdoren** [reply]
@binderth I use ix1600 on WiFi with Linux, and I'm very happy with it.

I run paperless-ngx on an 8GB Raspberry Pi 5 with 1TB USB NVME SSD on wired Ethernet and it works great with 4000 documents so far.

I expose my "consume" folder via SMB on the Linux side. Ix1600 can write to SMB share with current firmware.

WiFi is fully supported reliable.

I believe you need a PC or Mac for initial setup and firmware update over USB.  After that, you don't need USB any more.  I only use the manufacturer's "ScanSnap home" software for an occasional firmware update, which you can do even on WiFi.

I created a few profile buttons which show on ix1600 display, for 1 & 2 sided scanning, with and without 90 degree rotation.  This helps me minimize editing required later inside paperless.

I believe ix1600 has been superseded by ix2500 with similar specs.

**Reply by rajil** [reply]
@binderth I have had a brother scanner in the past  i think it was ADS-1600W model. The challenge with that was the paper used to become crooked while feeding in. One had to babysit the scanner to make sure that there were no misfeeds when scanning. I dont know how well is ADS-1800w in that respect. It seems that ix1600 has a hardware sensor to monitor misfeeds/double-feeds.

@kdoren Is it possible to invoke the scanner from the linux pc itself? 

There are times when i dont want to scan to paperless, instead i want to scan manually. I wonder how will that work with ix1600.

I wonder if ix2500 is a [downgrade](https://www.reddit.com/r/ScanSnap/comments/1lslmmz/ix2500_differences_from_ix1600/)? Seems it cannot write to SMB share.

**Reply by kdoren** [reply]
@rajil Current ix2500 firmware supports scanning to SMB share.  It can apparently also do auto-rotation when scanning to a network folder (which ix1600 cannot).

to use with Linux for something other than paperless, I'd create another SMB share, plus an additional profile button on the scanner that sends to it.

There is no equivalent to "ScanSnap home" for Linux.

https://www.pfu.ricoh.com/global/scanners/topics/news20251015-1/#:~:text=The%20iX2500%20now%20supports%20direct,a%20PC%20or%20other%20device.

**Reply by rajil** [reply]
@kdoren Thanks for the tip. Can it work with Nextcloud/webdav?

**Reply by kdoren** [reply]
@rajil It doesn't specifically support nextcloud/webdav.

I don't use either, but I believe nextcloud can be configured to monitor a folder, similar to the consume folder in paperless.

Also, being Linux, it shouldn't be hard to make a script or other process to monitor a folder and send to WebDAV.  Once the scanner deposits a file in your SMB shared directory, you can do whatever you want with it.

**Comment by github-Jan** [comment]
I use a Kyocera 5526cdw Multifunction Printer/Scanner.
+ supports Single pass Duplex scan 
+ supports scanning to different samba targets (e.g. consume folders)
+ works without PC
+ fast color laser printer 
o supports two favorites e.g. switching between duplex and single side scans
o medicore empty page detection (e.g. punched papers might not always be detected)
o sometime scans inclined
o configuration interface is not very good.
- Heats the printer before scanning (?) at least it smells like that.
- scanning many pages at once often results in missed pages or paper jams

Supports some scan to email/ftp which i never used.

My overall feedback:
okisch as a 4 in 1 model.
ok if you can some pages, not great if you want to scan 100 pages which are mixed single/dual side.


I think about buying a dedicated scanner if that supports a reliable easy scan part as i have about 30 paper folders i want/must scan.

My wish: 
Allow a single click scan target + settings 
e.g. easy: 
scan as double sided to folder Jan
scan as single sided to folder Jan
scan as double sided to folder Wife
scan as single sided to folder Wife
..

important for me is that i can easyly put many pages in there and scan everything without issues

**Comment by kdoren** [comment]
I'm really happy with the ScanSnap ix1600.  It's a beautiful design, and I like how the input tray can fold over to keep dust out of the sensor.  Paper handling is great, and so is scan quality.  I scan directly into an SMB share of the consume folder.

I run the scanner over WiFi (no USB after initial setup), it doesn't need a computer to scan to an SMB share.

I sometimes have documents where a few pages are in landscape mode, and I need to rotate those pages only.  Paperless-ngx cannot do this (or if it can, I don't know how).  In that case, I scan to the ScanSnap app on my Mac, which auto-rotates each page, then I save into the same SMB share. Easy.

**Reply by wduda** [reply]
Paperless is able to rotate pages when doing the OCR as long as it detects rotation being needed: https://docs.paperless-ngx.com/configuration/#PAPERLESS_OCR_ROTATE_PAGES

And then you can manually rotate the document using Actions inside paperless. Scanning into an app to rotate is only needed if you need to rotate just a part of the pages in a multipage pdf.

**Comment by hiranchaudhuri1** [comment]
I like this discussion, it has valuable information.
Keep in mind some more information has been collected in https://github.com/paperless-ngx/paperless-ngx/wiki/Scanner-&-Software-Recommendations

**Reply by wduda** [reply]
As an owner of a Fujitsu ix500 I already edited that list twice because wrong info was added, especially about being able to store in SMB folders - many responses seem to indicate people added a yes for that if the scanner was able to save to a SMB drive set up on a Windows PC but that defeats the point. Also there is no mention of where which user recommended a scanner or how they described their usecase, only that they apparently did at some point. So my trust in that list is rather low.

**Comment by andymarden** [comment]
Went round this direction recently myself. Settled on an epson ts-730n. Can be completely standalone, network connected (wired), double sided, fast, confirmed via built in web server, sent to email, ftp, sftp, smb, webdav - multi or single page pdf, one press button scanning for first 3 presets. Got it for £250 new.

40ppm bulk feed is a slight improvement on my 0.5ppm single sided sit there and manual scan each page on my flatbed scanner that's part of my printer🤣 

That mountain of years of paperwork is now starting to go down.

**Comment by schlegel11** [comment]
I can also recommend the Epson devices. I bought myself a DS-800WN. Not very cheap but totally worth it.
Standalone, wifi, gbit network, scan to mail (via own mail server), network drive support (smb, webdav, ftp...) etc.
I can configure the most directly on the device or I can also use the web ui, which is totally fine and usable.
I can configure 48 profiles e.g. paperless scan multi page, paperless scan single page... whatever I want.

Before that I tried also a ScanSnap ix2500 but its a joke, really. I mean the device/hardware seems to be pretty good but unfortunately the software decisions are so bad.
The device has a beautiful display which is only there to tell you that you can configure everything on your smartphone or computer ... that's it more or less apart a few settings.
So you cannot use this Scanner without a connected PC or smartphone. The best thing is that it is not possible at all to configure any kind of network drive (yet). I heard it was possible for the ix1600.
So you buy a 450$ device which is not capable to configure or use any kind of network drive. Wow. Just Wow.
Ah and of course there is no web ui or anything. Maybe there is an API because something is reachable via http but who cares.

**Comment by audiocrush** [comment]
Hi there.
In case someone wants a dirt-cheap fast and reliable scanner option, I have worked out a way to do it with old USB document scanners. I documented all of it on my blog and also try to explain briefly why I did it this way. Would be nice to hear some opinions on the writeup.
https://www.nerdsh.org/?p=394

**Comment by johnstephan1580-stack** [comment]
If you’re looking for a truly flexible, touch-screen [scanner](https://lambock.com/) that doesn’t tie you to a PC, I’d lean toward the Brother ADS series or the Fujitsu ix1600—they’re fast, reliable, and handle direct-to-cloud or network destinations really well. Your older ScanSnap experience shows the hardware is solid, but the software limitations can be frustrating, so exploring the newer models or Brother options makes sense.

**Comment by ulope** [comment]
I'm using a Brother ADS-2400N. It's perfect for my needs:

- Duplex
- One button operation
- Scans directly to SMB (also supports a whole bunch of other protocols)
- Can scan long receipts (no double feed recognition in this mode, so I set it to the second shortcut button)
- Very reliable
- Fast

**Comment by niels-przybilla** [comment]
The brother scanners do a solid job. And they can easily connect via S-FTP to paperless consume.
The scansnap ix1600 is the much better scanner for different formats and paper handling. But as far as I know, you always need a computer.

**Reply by tmaier** [reply]
Not true about ix1600. You may need a computer for the initial configuration, but then not at all. See my comment above where I compared ix1600 against a brother https://github.com/paperless-ngx/paperless-ngx/discussions/5042#discussioncomment-11285739

**Comment by mwrsa** [comment]
HP ScanJet Pro 4500fn1 here, and it's an absolute charm. I read about it beforehand and bought the precise model on purpose. It scans directly into the consume folder for me here, via LAN or WLAN. In my mobile office. Inside my camper van...

I could also quote @ulope here, because I could check the points on his list as well.

Overkill? Certainly. Inner nerd satisfied? Even more so :-D

**Comment by userbradley** [comment]
I've been using a `Brother ADS-2700W` and was able to scan every bit of post I've received since 2016 in an evening. Decent scanner and was not terribly priced on Ebay

**Comment by Bush-cat** [comment]
I recommend checking the scanner’s SANE standard implementation before buying, if possible. This can be useful if you want to avoid relying on the OEM software and drivers in the future.
I bought a Fujitsu fi-7160 to use directly over USB with my server instead of getting a Wi-Fi scanner or dealing with crappy drivers like my old Epson Printer. It worked great on the desktop, but when I tried the web UI Scanservjs, I had to override the scan dimensions because Fujitsu didn’t provide correct SANE sizes so scans were always cut off.
I think having a standard like SANE is great, but if I were buying another scanner, I'r try to research if there are any SANE ilnesses from the oems. The scanservjs docs mention some oems or you can go through github issues https://github.com/sbs20/scanservjs/blob/master/docs/12-recipes.md

**Reply by binderth** [reply]
that's ok, if your paperless-ngx server is right next to the scanner. In my case, the scanner is in the middle of the hallway, right next to the place, we put mail. So it's just "open the mail" - "put it in the scanner" for us. The server rack with the docker running paperless-ngx is in the basement. So it's not possible to connect them via USB.
But: paperless-ngx offers a variety of automated consuming options like email, hotfolders, etc. It shouldn't be a problem to configure whatever WiFi-scanner you have to either send emails or write directly on the inbox-consuming folder of paperless-ngx, if you can't physically connect the scanner to a server. Plus: you avoid all that fiddeling with serverside pre-processing of the scan.

**Reply by Bush-cat** [reply]
> that's ok, if your paperless-ngx server is right next to the scanner. In my case, the scanner is in the middle of the hallway, right next to the place, we put mail. So it's just "open the mail" - "put it in the scanner" for us. The server rack with the docker running paperless-ngx is in the basement. So it's not possible to connect them via USB. But: paperless-ngx offers a variety of automated consuming options like email, hotfolders, etc. It shouldn't be a problem to configure whatever WiFi-scanner you have to either send emails or write directly on the inbox-consuming folder of paperless-ngx, if you can't physically connect the scanner to a server. Plus: you avoid all that fiddeling with serverside pre-processing of the scan.

That's a nice use case as well. My server is just a minipc close to the desk and my mailbox is outside .But you can also use a usb scanner over wifi by using an sbc like a raspberry pi as scanservjs server.
I currently do the processing like ocr through paperless-ngx but I may extend my scanservjs with some pipelines in the future, I like having the option. I had to set a different brightness and contrast for the scanner tho so that text from the other side wouldn't be visible on a scan.

**Comment by github4soft** [comment]
Any recommendations for a scanner that ignores holes made by a hole punch on the back of a paper when scanning into a folder?
